code/utils.py

```diff
@@ -1,3 +1,18 @@
 """
 This file will contain helper methods for use across files
-"""
\ No newline at end of file
+"""
+
+# a dictionary that stores the mapping of email_id to id that is used to lookup user embeddings
+user_id_lookup = {}
+
+
+def get_userid(u):
+    """
+    gets the unique user_id for every user. Returns none if the mapping is not present
+    :param u:
+    :return:
+    """
+    if u in user_id_lookup:
+        return user_id_lookup[u]
```

Comment:
I'm guessing this is yet to be implemented.
Can we not use email ids directly to indeed the embeddings?

Reply:
nope ... nn.Embeddings expects a long tensor, it can't deal with strings